wolfkeymgr/mod_etsi.h

```diff
@@ -208,7 +208,8 @@ WOLFKM_API int wolfEtsiCalcTlsFingerprint(EtsiKeyType keyType,
 /* Build public name for key */
 WOLFKM_API int wolfEtsiKeyComputeName(EtsiKey* key);
 
-/* these are required if using multiple threads sharing the wolfSSL library for init mutex protection */
+/* these are required if using multiple threads sharing the wolfSSL library 
```

Comment:
This comment doesn't capitalize the first character, but the previous comment does.  Not sure which is "correct" but should pick one approach for consistency.